package.json

```diff
@@ -34,7 +34,10 @@
   "dependencies": {
     "commander": "^2.9.0",
     "datatype-expansion": "^0.2.3",
-    "js-yaml": "^3.6.1"
+    "deep-get-set": "^1.1.0",
+    "js-yaml": "^3.10.0",
```

Comment:
can we get rid of `js-yaml` altogether? i.e. does `yaml-ast-parser` provide equivalents to `yaml.safeLoad()` and `yaml.safeDump()`? if so let's use them and remove that dependency.

Reply:
I don't think yaml-ast-parser has this feature. -> https://github.com/mulesoft-labs/yaml-ast-parser/issues/1

Reply:
👌